pkg/thing/delivery/http/thing_proxy.go

```diff
@@ -231,8 +231,15 @@ func (p proxy) getPaginatedThings(authorization string) ([]*ThingProxyRepr, erro
 			p.logger.Error(err)
 			return nil, err
 		}
+
```

Comment:
In order to maintain code standardization throughout the project, I wouldn't add a  blank line between the 'defer' statement and `keepGoing`. It also seems that other projects such as [mainflux](https://github.com/mainflux/mainflux/blob/master/sdk/go/sdk.go) and [Go](https://github.com/golang/go/search?l=Go&q=defer) itself, seem to not add a line between the `defer` statement and the function it regards to.

Reply:
Thanks for advision @lcbm. Due to your comment I also remove another blank lines before `defer` statement in code and also change the commit message. Done!

Reply:
Awesome! :confetti_ball: